marketing/content/securesbom_vs_cosign.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "SecureSBOM vs Sigstore"
+description: "A side-by-side comparison between SecureSBOM and Sigstore across key criteria such as use cases, SBOM support, key management, and authentication."
+date: 2025-10-20
+draft: false
+---
+
+# SecureSBOM vs Sigstore
+
+A detailed comparison of **SecureSBOM** and **Sigstore**, two solutions addressing software supply chain integrity through signing and verification technologies.  
+While **Sigstore** focuses primarily on open-source ecosystems and OCI artifacts, **SecureSBOM** is built for enterprise-grade SBOM signing, verification, and compliance.
+
+---
+
+## 🧩 Primary Use Case
+
+| Tool | Description |
+|------|--------------|
+| **Sigstore** | Open Source Software, OCI-Focused (SBOMs attached as a container artifact). |
+| **SecureSBOM** | Business / Enterprise Software, purpose-built for SBOM signing and verification. |
+
+---
+
+## 📘 SBOM Support
+
+| Tool | Description |
+|------|--------------|
+| **Sigstore** | Treats all SBOMs as simple blobs. Supports **detached signatures only** for both SPDX and CycloneDX. |
+| **SecureSBOM** | Integrates directly with the **CycloneDX SBOM format** — supports **embedded signatures**, **property exclusion**, and also supports **SPDX detached signatures**. |
+
+---
+
+## 🔐 Key Management
+
+| Tool | Description |
+|------|--------------|
+| **Sigstore** | Ephemeral, short-lived keys/certificates from **Fulcio** with **OIDC authentication** — or manual key management (local or via cloud providers like GCP/AWS). |
+| **SecureSBOM** | **Automated** key lifecycle management using **cloud HSMs**. Ideal for enterprise compliance and long-term trust. |
+
+---
+
+## 🪪 Authentication
+
+| Tool | Description |
+|------|--------------|
+| **Sigstore** | OIDC (for keyless signing) or manual authentication for self-managed keys. |
+| **SecureSBOM** | Authenticated via **API Key**, enabling fine-grained access control and multi-tenant isolation. |
+
+---
+
+## 🔍 Transparency
+
+| Tool | Description |
+|------|--------------|
+| **Sigstore** | Public Record via **Rekor** (Certificate Transparency Logs). Verification relies on transparency records and public keys. |
```

Comment:
Note, there are two transparency logs in the Sigstore ecosystem.  Rekor, which stores the metadata associated with what has been signed, and a certificate transparency log, which is  a public record of all certificates issued.

Perhaps simplify this to:

Public record via public transparency logs.  Verification relies on public keys and the public transparency logs.